### PR TITLE
feat: show meta framework name on server build success

### PR DESCRIPTION
**Pull request by ayoayco**
### 🔗 Linked issue

#1954 

### ❓ Type of change

- [ ] 📖 Documentation (updates to the documentation, readme, or JSdoc annotations)
- [ ] 🐞 Bug fix (a non-breaking change that fixes an issue)
- [ ] 👌 Enhancement (improving an existing functionality like performance)
- [x] ✨ New feature (a non-breaking change that adds functionality)
- [ ] 🧹 Chore (updates to the build process or auxiliary tools and libraries)
- [ ] ⚠️ Breaking change (fix or feature that would cause existing functionality to change)

### 📚 Description

Let me know if this is not the direction the project wants, I just assumed it is after seeing the new `framework` configuration :)
 
This PR will show the framework name and version on dev server's build success.

### 📝 Checklist

- [x] I have linked an issue or discussion.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by pi0** [comment]
This is nice idea! Only we could show Both Nitro version and framework versions.

**Comment by ayoayco** [comment]
Alright! Will push the change in a bit :)

File: src/build.ts [hunk 1]
```diff
@@ -484,7 +484,7 @@ function startRollupWatcher(nitro: Nitro, rollupConfig: RollupConfig) {
         nitro.hooks.callHook("compiled", nitro);
         const { framework } = nitro.options;
         nitro.logger.success(
-          `${framework.name} v${framework.version} built`,
+          `Nitro Server v${nitroVersion} built for ${framework.name} v${framework.version}`,
```
Comment (pi0) [inline src/build.ts:487]: This is good! We only now to check if `framework.name !== "nitro"` to avoid showing `Nitro server v1 build for Nitro v1` :D

**Comment by pi0** [comment]
I like the output but looking at screenshot, seems too verbsoe no? We might try to colorize indeed and gray out versions or maybe just show names to be shorter?

**Comment by pi0** [comment]
On an unrelated note: If you are on Discord would you DM me please? Would love you to have you in UnJS server :)

**Comment by ayoayco** [comment]
I don't know which discord it is @pi0 😬

**Comment by ayoayco** [comment]
> I like the output but looking at the screenshot, seems too verbose no? We might try to colorize indeed and gray out versions or maybe just show names to be shorter. (ideas welcome to try different modes)

I will play around more with the colors then let you know when I'm done :)

**Comment by ayoayco** [comment]
@pi0 Here's the latest screenshots...

When no framework info or `showBuildSuccess` is `false`
Screenshot 2023-11-27 at 18 28 32

When framework info & `showBuildSuccess` is `true`
Screenshot 2023-11-27 at 18 29 01

When `showBuildSuccess` is `'verbose'`
Screenshot 2023-11-27 at 18 29 20

**Comment by ayoayco** [comment]
All sounds good! :)

**Pull request merged**